vhd/packer/configure-windows-vhd.ps1

```diff
@@ -92,7 +92,7 @@ function Get-FilesToCacheOnVHD {
             "https://github.com/microsoft/SDN/raw/master/Kubernetes/windows/helper.psm1",
             "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1",
             "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
-            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.3.zip"
+            "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.4.zip"

```

Comment:
Should we keep v.0.03 here too until we get a new VHD published / released with these changes?

Reply:
Would it affect anything since these are only used during vhd publish? I think it is easier to remember to update now and than when building vhd at later time

Reply:
oh sorry I see, you want to leave both .3 and .4 in the cache. Got it.